deployment/copilot/README.md

```diff
@@ -49,11 +49,20 @@ See Replayer explanation [here](../../TrafficCapture/trafficReplayer/README.md#a
 
 The migration solution has support for running multiple Replayer services simultaneously, such that captured traffic from the Capture Proxy (which has been stored on Kafka) can be replayed on multiple different cluster configurations at the same time. These additional independent and distinct Replayer services can either be spun up together initially to replay traffic as it comes in, or added later, in which case they will begin processing captured traffic from the beginning of what is stored in Kafka.
 
-A **prerequisite** to use this functionality is that the migration solution has been deployed with the `devDeploy.sh` script, so that necessary environment values from CDK resources like the VPC, MSK, and target Domain can be retrieved for additional Replayer services
+A **prerequisite** to use this functionality is that the migration solution has been deployed with the `devDeploy.sh` script, so that necessary environment values from CDK resources like the VPC, MSK, and EFS volume can be retrieved for additional Replayer services
 
-To spin up another Replayer service, a command similar to below can be ran where `id` is a unique label to apply to this Replayer service and `target-uri` is the accessible endpoint of the target cluster to replay traffic to.
+To try this scenario out, an additional Opensearch Domain target cluster can be created in the existing VPC by running a series of commands like below
```

Comment:
To test this scenario, you can create an additional OpenSearch Domain target cluster within the existing VPC by executing the following series of commands: